### PR TITLE
nicer README

### PR DESCRIPTION
**Pull request by greedo**
- formatted README using ReStructured for nicer looking docs.
- added PyPi version badge

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    def app():
+        app = create_app()
+        return app
+
+Install the extension with dependencies and go::
+
+    $ pip install pytest-flask
+    $ py.test
+
+***Found a bug? Have an issue?***
```
Comment (vitalk) [inline README.rst]: I suppose the 2nd-level header is appropriate.

Comment (greedo) [inline README.rst]: @vitalk agreed, it could even say `Contributing` similar to the PyPi page

Comment (vitalk) [inline README.rst]: :+1:

**Comment by vitalk** [comment]
Thanks you for your contribution!

**Pull request merged**